### PR TITLE
fix: add nuget package for all remaining targets

### PR DESCRIPTION
**Pull request by khooinguyeen**

### CODE REVIEW

File: TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj [hunk 1]
```diff
@@ -115,6 +115,9 @@
     <BundleResource Include="Resources\Fonts\uno-fluentui-assets.ttf" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="ToastNotification.Uno">
+      <Version>0.0.0-preview-18-835992017</Version>
```
Comment (trungnt2910) [inline TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj:119]: The iOS version is not released yet, but well... it should be here, so I'd say that's fine?

File: TimetableApp/TimetableApp.macOS/TimetableApp.macOS.csproj [hunk 2]
```diff
@@ -48,9 +53,12 @@
...
     <LinkMode>SdkOnly</LinkMode>
-    <HttpClientHandler></HttpClientHandler>
-    <XamMacArch></XamMacArch>
-    <NoWarn></NoWarn>
+    <HttpClientHandler>
+    </HttpClientHandler>
+    <XamMacArch>
+    </XamMacArch>
+    <NoWarn>
+    </NoWarn>
```
Comment (trungnt2910) [inline TimetableApp/TimetableApp.macOS/TimetableApp.macOS.csproj:61]: Remove these changes, it makes the commit history messy.

File: TimetableApp/TimetableApp.macOS/TimetableApp.macOS.csproj [hunk 3]
```diff
@@ -61,6 +69,9 @@
     <Reference Include="System.Memory" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="ToastNotification.Uno">
+      <Version>0.0.0-preview-18-835992017</Version>
+    </PackageReference>
```
Comment (trungnt2910) [inline TimetableApp/TimetableApp.macOS/TimetableApp.macOS.csproj:74]: Only keep these lines.

**Review by trungnt2910 — CHANGES_REQUESTED** [review]
You should revert all changes to all files, except the blocks that say "PackageReference"

File: TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">iPhoneSimulator</Platform>
-    <ProjectGuid>e600c9b2-b71d-4bf1-a7fc-64c9220a4b48</ProjectGuid>
+    <ProjectGuid>{E600C9B2-B71D-4BF1-A7FC-64C9220A4B48}</ProjectGuid>
```
[the remark this answers is not in the document]
Comment (khooinguyeen) [reply TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj]: okayy

File: TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj (hunk 1 above)
Comment (khooinguyeen) [reply TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj:119]: you said the ToastNotification.uno is for all other platforms :))

**Comment by khooinguyeen** [comment]
I forgot to rename of the commit :)) It has to be `fix: cleaned up the mess in the .csproj files`

**Comment by trungnt2910** [comment]
> I forgot to rename of the commit :)) It has to be `fix: cleaned up the mess in the .csproj files`

There is a way to rename commits.
try
`git rebase -i HEAD~num` where `num` is the position of the commit, starting from HEAD.
In this case, this is 2.
so `git rebase -i HEAD~2`
Then, a code editor will appear (most likely to be VS Code), then for each commit you want to rename, remove `pick` and change it to `reword`.
After that, follow the instructions to type the new names.
Finally, you **must** force push to your branch, by using `git push -f`.

There is a detailed guide [here](https://linuxize.com/post/change-git-commit-message/#changing-an-older-or-multiple-commits), you may want to read it.

**Comment by khooinguyeen** [comment]
urghh

**Comment by khooinguyeen** [comment]
finally 😐

File: TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj (hunk 1 above)
Comment (trungnt2910) [reply TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj:119]: Yeah it's alright.

**Comment by trungnt2910** [comment]
Great, thanks for your contribution.

**Review by trungnt2910 — APPROVED** [review]

**Pull request merged**